contrib/windows/msys_build.sh

```diff
@@ -203,7 +203,7 @@ fi
 echo 'FORCE_ASSERTIONS = 1' >> Make.user
 
 cat Make.user
-make -j3 VERBOSE=1 install
+make -j3 VERBOSE=1
```

Comment:
I thought that this file was supposed to be a general-purpose msys build helper, not specific to appveyor, so I was trying to minimize the testing-specific work it was doing.